pkg/providers/ovirt/mapper/mapper_test.go

```diff
@@ -29,6 +29,43 @@ var (
 	osFinder = mockOsFinder{}
 )
 
+var _true bool = true
+var _false bool = false
+
+var _ = Describe("Test mapping virtual machine bios type", func() {
+	BeforeEach(func() {
+		findOs = func(vm *ovirtsdk.Vm) (string, error) {
+			return "linux", nil
+		}
+	})
+
+	table.DescribeTable("to smm feature ", func(biostype ovirtsdk.BiosType, smm *bool) {
+		vm := createVMGeneric(ovirtsdk.VMAFFINITY_USER_MIGRATABLE, true, biostype)
+
+		mappings := createMappings()
+		credentials := mapper.DataVolumeCredentials{
+			URL:           "any-url",
+			SecretName:    "secret-name",
+			ConfigMapName: "config-map",
+		}
+		namespace := "the-namespace"
+		mapper := mapper.NewOvirtMapper(vm, &mappings, credentials, namespace, &osFinder)
+		vmSpec, _ := mapper.MapVM(&targetVMName, &kubevirtv1.VirtualMachine{})
+
+		if *smm {
+			Expect(vmSpec.Spec.Template.Spec.Domain.Features).ToNot(BeNil())
```

Comment:
That check can be moved before if - the other branch assumes that `vmSpec.Spec.Template.Spec.Domain.Features` are not nil.